backend/otel/otel.go

```diff
@@ -339,9 +339,9 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	var messages []*kafkaqueue.Message
 	for projectID, traceMetrics := range projectTraceMetrics {
 		for sessionID, metrics := range traceMetrics {
```

Comment:
same bug here?

Reply:
this was different but could cause duplicate metrics as the `messages` list could be submitted to kafka multiple times

Reply:
what about the `&projectID` reference on line 349?